### PR TITLE
fix: enable builds on Windows

### PR DESCRIPTION
**Pull request by padamstx**
## PR summary
This PR modifies the project to allow developers to build and test the project on Windows.

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by padamstx** [comment]
Updated this PR to completely remove the pylint.sh and .bat files. Instead, I updated the Makefile so that the "lint" and "lint-fix" targets invoke pylint directly instead of via the scripts.

File: update_service.md [hunk 1]
```diff
@@ -75,10 +78,13 @@ git pull
...
 
-# source in the resulting "activate" script to use the virtual environment
+# Activate the virtual environment
+# On Linux/MacOS source in the "activate" script created by the "venv" module:
 . venv/bin/activate
+# On Windows, run the "activate" script created by the "venv" module:
+venv/scripts/activate
 
-# Install dependencies, run unit-tests and link-check the project
+# Install dependencies, run unit-tests and lint-check the project
```
Comment (pyrooka) [inline update_service.md:87]: 😮

**Review by pyrooka — APPROVED** [review]
Looks good!

File: update_service.md (hunk 1 above)
Comment (dpopp07) [reply update_service.md:87]: Chief typo finder! 👑

**Review by dpopp07 — APPROVED** [review]
LGTM

File: update_service.md (hunk 1 above)
Comment (padamstx) [reply update_service.md:87]: :joy:

**Pull request merged**